README.rst

```diff
@@ -32,6 +32,16 @@ Installation
 See doc/install.txt in the source code archive.
 Python 2.7.2 or later is needed.
 
+
+Install using Docker
+---------------------
+You can try linkchecker in an isolated enviroment without changing any local files or have to install all the dependencies via a Docker Container:
+
+``docker pull ellerbrock/linkchecker``
+
+https://hub.docker.com/r/ellerbrock/link-checker/
```

Comment:
Also: what should the user do after running the docker pull?  I assume it'll be something like `docker run linkchecker /usr/bin/linkchecker http://example.com`?